src/analysis/classic/warlock/demonology/guide/DoomguardSection.tsx

```diff
@@ -0,0 +1,62 @@
+import { SpellLink } from 'interface';
+import SPELLS from 'common/SPELLS/classic';
+import { GuideProps, Section, SubSection } from 'interface/guide';
+import CombatLogParser from '../CombatLogParser';
+import { FoundationHighlight as HL } from 'interface/guide/foundation/shared';
+
+export function DoomguardSection({ modules, events, info }: GuideProps<typeof CombatLogParser>) {
+  function SnapshotQuality() {
+    if (modules.doomguard.doomguardSummonData.doomguard.summonTimestamp !== 0) {
+      return (
+        <>
+          Snapshot quality:
+          {modules.doomguard.snapshotQualityEntries.map((entry) => {
+            return entry.snapshotSummary;
+          })}
+          {modules.doomguard.snapshotAdvice}
+        </>
+      );
+    } else {
+      return <></>;
+    }
+  }
+
+  return (
+    <Section title="Summon and buff your doomguard">
+      <p>
+        <SpellLink spell={SPELLS.SUMMON_DOOMGUARD} /> is your most powerful damage cooldown, and
+        contributes to a large portion of your DPS. <br />
+        In most of fights, you can use this cooldown only once per fight, due to its 10 mins
+        cooldown, and it lasts for 1:05 mins (45s base duration + 20s thanks to{' '}
+        <SpellLink spell={SPELLS.ANCIENT_GRIMOIRE} /> talent)
+        <br />
+      </p>
```

Comment:
don't use `<br/>` for spacing or manual line-breaks. it breaks on differing screen sizes (also grammar)

```suggestion
      <p>
        <SpellLink spell={SPELLS.SUMMON_DOOMGUARD} /> is your most powerful damage cooldown, and
        contributes a large portion of your DPS. 
        Due to the 10 minute cooldown, your goal on most fights is to get <strong>one good cast</strong> for the full 1m 5s duration.
      </p>
```